.github/workflows/lib-injection.yml

```diff
@@ -0,0 +1,114 @@
+      run: |
+        cp ./lib-injection/* ${{steps.assets.outputs.artifacts_path}}
+
+        # Temporarily create a placeholder linux-musl-arm directory
+        mkdir -p ${{steps.assets.outputs.artifacts_path}}/linux-musl-arm
+        touch ${{steps.assets.outputs.artifacts_path}}/linux-musl-arm/placeholder.txt
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Generate libc image tags
+      id: lib-injection-tags
+      uses: DataDog/system-tests/lib-injection/docker-tags@main
+      with:
+        init-image-name: 'dd-lib-dotnet-init'
+        main-branch-name: 'master'
+
+    - name: Generate musl image tags
+      id: generate-musl-tags
+      shell: bash
+      run: |
+        # Update first IFS character to ',' to use as our delimiter
+        IFS=','
+
+        # Read the csv tag-names into an array variable and append '-musl' to each tag
+        read -r -a array <<< "${{ steps.lib-injection-tags.outputs.tag-names }}"
+        array=( "${array[@]/%/-musl}" )
+        MUSL_TAG_NAMES=${array[@]}
+
+        echo "Generated musl tag names '$MUSL_TAG_NAMES'"
+        echo "musl-tag-names=$MUSL_TAG_NAMES" >> $GITHUB_OUTPUT
+
+    - name: Login to Docker
+      run: docker login -u publisher -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
+
+    - name: Docker Build linux-x64 image
+      uses: docker/build-push-action@v3
+      with:
+        push: true
+        tags: ${{ steps.lib-injection-tags.outputs.tag-names }}
+        platforms: 'linux/amd64' # for windows, we can run windows/amd64,windows/386,windows/arm64
+        context: ${{steps.assets.outputs.artifacts_path}}
+        build-args: |
+          LINUX_PACKAGE=datadog-dotnet-apm-${{steps.versions.outputs.version}}.tar.gz
+
+    - name: Docker Build linux-arm64 image
+      uses: docker/build-push-action@v3
+      with:
+        push: true
+        tags: ${{ steps.lib-injection-tags.outputs.tag-names }}
+        platforms: 'linux/arm64/v8'
+        context: ${{steps.assets.outputs.artifacts_path}}
+        build-args: |
+          LINUX_PACKAGE=datadog-dotnet-apm-${{steps.versions.outputs.version}}.arm64.tar.gz
+
+    - name: Docker Build linux-musl-x64 image
+      uses: docker/build-push-action@v3
+      with:
+        push: true
+        tags: ${{ steps.generate-musl-tags.outputs.musl-tag-names }}
+        platforms: 'linux/amd64'
+        context: ${{steps.assets.outputs.artifacts_path}}
+        build-args: |
+          LINUX_PACKAGE=datadog-dotnet-apm-${{steps.versions.outputs.version}}-musl.tar.gz
+
+    - name: Docker Build linux-musl-arm64 image
+      uses: docker/build-push-action@v3
+      with:
+        push: true
+        tags: ${{ steps.generate-musl-tags.outputs.musl-tag-names }}
+        platforms: 'linux/arm64/v8'
+        context: ${{steps.assets.outputs.artifacts_path}}
+        build-args: |
+          LINUX_PACKAGE=linux-musl-arm
```

Comment:
Shouldn't it be ?
```suggestion
          LINUX_PACKAGE=LINUX_PACKAGE=datadog-dotnet-apm-${{steps.versions.outputs.version}}-musl-arm64.tar.gz
```

Reply:
We don't yet have a linux-musl-arm64 binary, so I've added comments to clarify that in 3bb0dd4be93cde292977a97c599f2f6401cd5d3f